content/docs/user-guide/gems/development/creating/_index.md

```diff
@@ -8,12 +8,25 @@ You can develop your own independent modules to distribute and build as Open 3D
 bundle of code and/or assets, a manifest file, a CMake build file, and an optional display icon for project configuration
 tools. This is all that's needed to ship a Gem.
 
-Although you can create a Gem manually by creating all of the files yourself, it's recommended to use the `o3de` tool:
+Although you can create a Gem manually by creating all of the files yourself, it's recommended to use the `o3de` tool located in the `<engine>/scripts` directory:
+
+{{< tabs name="Create a Gem" >}}
+{{% tab name="Windows" %}}
 
 ```cmd
-o3de create-gem -gp <path to create gem at>
+<engine>/scripts/o3de.bat create-gem -gp <path to create gem at>
```

Comment:
Using a forward slash will fail in the Windows command prompt. Prefer back slash for Windows-specific command examples.
```suggestion
<engine>\scripts\o3de.bat create-gem -gp <path to create gem at>
```